### DISCUSSION TITLE
Weekly Progress of DeePMD-kit 20220822

### DISCUSSION THREAD

**Discussion by HuangJiameng** [deepmodeling/deepmd-kit#1871] [Show and tell]
# What's new

**DeePMD-kit v2.1.4 release**
### Enhancements:
- #1800 
- #1804 
- #1744

### Bug fixings:
- #1862 
- #1802 
- #1811

Check out https://github.com/deepmodeling/deepmd-kit/releases/tag/v2.1.4 to learn more about DeePMD-kit v2.1.4.

# Discussion

- #1860
The author is wondering in what cases must fp64 be used to train the model in order to get enough accuracy. Awaiting answers.

- #1859
If you find the calculated free energies abnormal, you may follow the section 9 of this work http://iopscience.iop.org/0953-8984/20/15/153101 to check the calculation of free energies.

- #1761 
UnsatisfiableError occurred when installing DeePMD-kit 2.1.1(gpu version). This problem is said to exist in DeePMD-kit 2.1.3 too. 

- #1822
1. One can output the Tensorboard trace for inference by writing a sample Python script with input data and use the TensorFlow Profiler to trace the bottlenecks. 
2. [dpdata](https://github.com/deepmodeling/dpdata/blob/master/dpdata/md/rdf.py) can help you get the Radial Distribution Function of your model.
#1861 and #1842 are also related to this topic.

- #1869
One may check the packages (conda list, pip list) and environment variables (env) to see if anything is wrong if RuntimeError `This deepmd-kit package is inconsistent with TensorFlow Runtime` occurs.

**Comment by wanghan-iapcm** [comment]
We may want to mention the release of v2.1.4
